11ty/definitions/xenophobia.md

```diff
@@ -0,0 +1,22 @@
+---
+title: xenophobia
+slug: xenophobia
+defined: true
+excerpt:  hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+speech: noun
+flag:
+  level: warning
+  text: content warning
+---
+
+hatred of and discrimination against people from other countries, their culture, and/or their religion that is rooted in the fear of people who are "different"
+
+## Issues
+
+Xenophobia when broken down translates to the "fear of strangers." It stems from a psychological point of view that there is an "ingroup" and an "outgroup" that exist within a community, which is known as [ingroup-outgroup bias](http://psychology.iresearchnet.com/social-psychology/group/ingroup-outgroup-bias/). Whether it's conscious or unconscious, ingroup-outgroup bias can lead to harmful results, including xenophobia.
+
+## Impact
+
+Xenophobia can perpetuate physical, emotional, and psychological harm to those who look "different," which usually include people of color, immigrants, and minorities of a country. This can also extend harm to those who look similar to those that are perceived as "different." Xenophobia can further result in implicit bias, harmful public policies, hate crimes, and genocide.
+
+Xenophobia exists in many counties, including America. Some notable examples of [xenophobia in America](https://www.thoughtco.com/definition-of-xenophobia-and-examples-2834608) include the Chinese Inclusion Act of 1882 that prohibited Chinese immigrant laborers to enter America; Japanese-American internment camps that were created after the Pearl Harbor attack in 1941 under the guise that Japanese people in America were possible threats; and the Muslim Ban passed in 2017 that bans immigrants from specific Muslim-majority countries to enter America.
```

Comment:
It's implicitly said here, but I think it would bear to note that xenophobia  can manifest as both interpersonal bias (via microaggressions) as well as systemically (the examples you have here).

Reply:
I'd also love a note that a term I'm starting to see more in favour is -misia, so xenomisia rather than xenophobia.  In some interpretations, xenophobia can reinforce the idea that fear is valid, whereas it's really the hateful response that comes out that causes the issues laid here.